### ISSUE TITLE
igraph_setup.R Not Found

### ISSUE THREAD

**Issue by DarioS** [libscran/scrapper#3]
Perhaps it needs to be added to the repository by `git add`.
```
* installing *source* package ‘scrapper’ ...
** using staged installation
Fatal error: cannot open file 'igraph_setup.R': No such file or directory
```

**Comment by LTLA** [comment]
well, it's already there. works fine on my mac.

**Comment by DarioS** [comment]
May you try `devtools::install_github("libscran/scrapper")`?

**Comment by LTLA** [comment]
Fixed by 2f1dc4deec79f52c652a03cba89ecbef18d25721.
